<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkingPropertiesDialog</class>
 <widget class="QDialog" name="NetworkingPropertiesDialog">
  <property name="windowTitle">
   <string>Configure IP Addresses</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="blurbLabel">
     <property name="text">
      <string>Configure IP addresses for this host or pool.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="tellMeMoreLabel">
     <property name="text">
      <string>Tell me more...</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="leftPanel">
      <layout class="QVBoxLayout" name="leftLayout">
       <item>
        <widget class="VerticalTabWidget" name="verticalTabs"/>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Add IP address</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="contentPanel"/>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VerticalTabWidget</class>
   <extends>QListWidget</extends>
   <header>widgets/verticaltabwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NetworkingPropertiesDialog</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NetworkingPropertiesDialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>
